### PR TITLE
Add "Linux Command" extension.

### PR DESCRIPTION
**Pull request by jaywcjlove**
## Description

Search Linux command [Chinese](https://github.com/jaywcjlove/linux-command) documentation from Raycast.

## Screencast

image

image

image

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `media` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @jaywcjlove 👋 

Thanks for your contribution 💯 

At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

Data from source may contain Chinese though 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by jaywcjlove** [comment]
@pernielsentikaer This is a plugin for searching Chinese linux command documentation.

**Comment by pernielsentikaer** [comment]
Hi @jaywcjlove 👋 

We don't allow chinese in any script files, so lines like this:

`"description": "Linux命令大全搜索工具，内容包含Linux命令手册、详解、学习、搜集。",` needs to be written to something like `"description": "Search through linux commands and get detailed information in Chinese"`

Let me know if you have any questions, you can find me in [Slack](https://raycastcommunity.slack.com/) if you want to chat with me 💪

**Comment by jaywcjlove** [comment]
@pernielsentikaer thx! I changed the Chinese information in the script file.

File: extensions/linux-command/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "title": "Linux Command",
+  "description": "Search through linux commands and get detailed information in Chinese.",
+  "icon": "command-icon.png",
+  "author": "jaywcjlove",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Linux Command",
+      "subtitle": "搜索 Linux 命令",
```
Comment (pernielsentikaer) [inline extensions/linux-command/package.json:13]:
```suggestion
      "title": "Search Linux Command",
```

File: extensions/linux-command/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "linux-command",
+  "title": "Linux Command",
```
Comment (pernielsentikaer) [inline extensions/linux-command/package.json:4]:
```suggestion
  "title": "Search Linux Commands",
```

File: extensions/linux-command/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  return (
+    <List.Item
+      title={props.name}
+      subtitle={props.detail}
+      icon={Icon.Terminal}
+      actions={
+        <ActionPanel>
+          <Action.OpenInBrowser url={weburl} />
+          <Action.CopyToClipboard title="Copy Detail URL" content={weburl} />
+          <Action.Push
```
Comment (pernielsentikaer) [inline extensions/linux-command/src/index.tsx:40]: Missing an icon

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @jaywcjlove 👋 

I left some feedback on the missing parts, we're soon ready to merge 🚀 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/linux-command/package.json (hunk 1 above)
Comment (jaywcjlove) [reply extensions/linux-command/package.json]: Modification has been completed.

File: extensions/linux-command/src/index.tsx (hunk 3 above)
Comment (jaywcjlove) [reply extensions/linux-command/src/index.tsx:40]: Modification has been completed.

**Comment by jaywcjlove** [comment]
@raycastbot Modification has been completed.

**Review by pernielsentikaer — APPROVED** [review]
Hi @jaywcjlove 👋 

Thanks for the corrections, it was much better 💪 

Approved 🚀

**Pull request merged**